src/components/types/ApiRegistration.ts

```diff
@@ -0,0 +1,66 @@
+// subset of personal details that can be sent in POST requests
+export type CorePersonalDetails = {
+  first_name: string;
+  last_name: string;
+  nickname: string;
+  sex: string;
+  telephone: string;
+  telephone_opt_in: boolean;
+  language: string;
+  occupation: string;
+  age_group: string;
```

Comment:
`age_group` field type should be integer type it was REST API bug.

Reply:
Updated type to `number`.